roles/ceph-osd/tasks/main.yml

```diff
@@ -27,6 +27,21 @@
   # Hard code this so we will skip the entire file instead of individual tasks (Default isn't Consistent)
   static: False
 
+- name: create lvm OSDs with ceph-volume
```

Comment:
I'll fix the capital letters.

Actually, when reading logs from QE I've wished that includes had a name because it's sometimes hard to tell what file was being 'skipped' when the include isn't needed. I'd be +1 to requiring names for any includes going forward.